ontologies/works.ttl

```diff
@@ -0,0 +1,344 @@
+
+@prefix ww: <TBC/works> . 
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl:     <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+
+TBC
```

Comment:
This file currently is not currently valid (i.e. it's not machine readable). I think you can safely remove the TBC here.